decredPoS/decredPoS_script.md

```diff
@@ -37,13 +35,15 @@ After the update threshold is met voting will begin on the *first block* of the
 
 Each rule change interval spans 8,064 blocks (or about 4 weeks) during which votes for any active rule change processes are collected.  A revote occurs if 90% of votes collected during the entire interval abstain.  Else, if 75% of non-abstaining votes signal yes, then the proposed rule changes will be activated at the end of the *next* rule change interval.
 
-After a ticketholder's votes has been included in a block and a 256 block immaturity phase has passed, the price of the voted ticket is fully refunded and a portion of the Decred block reward is credited to their account as compensation for their active participation.
+After the ticketholder's votes has been included in a block and a 256 block immaturity phase has passed, the price of the voted ticket is fully refunded and a portion of the Decred block reward is credited to their account as compensation for their active participation.
 
 On-chain voting allows stakeholders to govern the chain, but what about governance of the project itself?
 
 While their live tickets are waiting in the ticket pool, ticketholders have access to proposal voting through Politeia, Decred's chain-anchored proposal system. Through Politeia anyone can shape the future of Decred by proposing new ideas for stakeholder consideration in an off-chain yet cryptographically verifiable manner.
 
-To begin participating in the governance of Decred, download Decrediton at Decred.org and up your stake by purchasing decred at any one of the many supporting exchanges.
+To begin participating in the governance download Decred's offical wallet Decrediton at Decred.org.
+
+You can up your stake by purchasing decred at any one of the many supporting exchanges.
```

Comment:
To begin participating in the governance system of Decred, download the Decrediton wallet at Decred.org. (avoid using official)

I'm also doubtful whether the phrase "up your stake" will be clear. We have to write these videos with non-native speakers and newcomers in mind as well